test/helper.rb

```diff
@@ -31,8 +32,14 @@
 require 'fluent/log'
 require 'fluent/test'
 require 'minitest/autorun'
-require 'webmock/test_unit'
 require 'vcr'
+require 'ostruct'
+require 'fluent/plugin/filter_kubernetes_metadata'
+require 'fluent/test/driver/filter'
+require 'kubeclient'
+
+require 'webmock/test_unit'
+WebMock.disable_net_connect!
```

Comment:
this modifies the global environment, so needs to be in the helper or running tests in a different order / single test leads to weird behavior